src/scripts/main.js

```diff
@@ -1 +1,20 @@
 'use strict';
+
+const colectionList = document.querySelector('.gallery__list');
+const imegLarge = document.querySelector('.gallery__large-img');
+
+colectionList.addEventListener('click', (E) => {
```

Comment:
variables should be in camelCase. If var is only one symbol, it should be in lower case
```suggestion
colectionList.addEventListener('click', (e) => {
```